lib/nameifier.go

```diff
@@ -1,11 +1,9 @@
-package main
+package lib
```

Comment:
This will be the name it gets imported by in consuming code. Probably better to call the package `nameifier`.

Reply:
if we call the package `nameifier` then when you `go build` it fails to compile the binary in the same directory?

Reply:
`go build: build output "nameifier" already exists and is a directory`